lib/octicons_gem/lib/octicons_v2/octicon_v2.rb

```diff
@@ -5,14 +5,11 @@ class OcticonV2
 
     def initialize(symbol, options = {})
       @symbol = symbol.to_s
-      if octicon = OcticonsV2::OCTICON_SYMBOLS[@symbol]
-
-        @path = octicon["path"]
-        @width = octicon["width"].to_i
-        @height = octicon["height"].to_i
-
-        @keywords = octicon["keywords"]
-
+      if octicon = get_octicon(@symbol, options)
+        @path = octicon[:path]
```

Comment:
Is there a reason we're switching to symbol keys? It might not matter in this case, but switching key formats has bitten us before, and I'd rather minimize the surface area of this change.

Reply:
We're now indexing into the hash table returned by the `get_octicon` function instead of the JSON object which had string keys. I can change the `get_octicon` function to return a hash with string keys if you think that's better.